lib/rails/generators/anycable/setup/docker/docker_generator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module AnyCableRailsGenerators
+  module Setup
+    # Generator to help set up AnyCable in Docker environment
+    class DockerGenerator < ::Rails::Generators::Base
+      namespace "anycable:setup:docker"
```

Comment:
Let's put these scripting into `anycable:setup:dev:...` namespace, 'cause it might be confused with production configuration.

Reply:
First of all, no one in their right mind won't execute this generator in production.
Also, almost nothing stops to use these AnyCable Compose services in production when deploying to Docker Swarm.